compiler/plugins/target/ROCM/ROCMTarget.cpp

```diff
@@ -267,7 +275,8 @@ class ROCMTargetBackend final : public TargetBackend {
   // ones). Inspired by code section in
   // https://github.com/iree-org/iree/blob/main/compiler/plugins/target/CUDA/CUDATarget.cpp
   static void optimizeModule(llvm::Module &module,
-                             llvm::TargetMachine &targetMachine) {
+                             llvm::TargetMachine &targetMachine,
+                             const llvm::ArrayRef<std::string> &passPlugins) {
```

Comment:
```suggestion
                            ArrayRef<std::string> passPlugins) {
```

'view' objects are typically just pointer + length and thus we prefer to copy them